src/app/login/utils.js

```diff
@@ -0,0 +1,29 @@
+import * as Yup from 'yup'
+
+export const getAlumniValidationSchema = () => Yup.object({
+  names: Yup.string().required('Ingresa tu nombre'),
+  paternalSurname: Yup.string().required('Ingresa tu apellido paterno'),
+  maternalSurname: Yup.string().required('Ingresa tu apellido materno'),
+  enrollmentId: Yup.number().required('Ingresa tu matricula').positive().integer(),
+  currentGroup: Yup.number().required('Ingresa tu turno').positive().integer(),
+  shift: Yup.string().required('Ingresa tu turno').oneOf(['Matutino','Vespertino'])
+})
+
+export const getEmailValidationSchema = () => Yup.object({
```

Comment:
Email? si es el que hace referencia al profesor cambialo por

```suggestion
export const getProfessorLoginValidationSchema = () => Yup.object({
```